src/base/geom/point.h

```diff
@@ -182,6 +181,10 @@ struct Point
 	[[nodiscard]] Point leftNormal() const { return Point{y, -x}; }
 
 	[[nodiscard]] Point rightNormal() const { return Point{-y, x}; }
+
+	consteval static auto members() {
+		return std::tuple{&Point::x, &Point::y};
+	}
```

Comment:
is it necessary? It can be aggregate initializable.

Reply:
Compiles, but not working.